### ISSUE TITLE
low contrast / accessibility problem for map clusters

### ISSUE THREAD

**Issue by rittermo** [liqd/adhocracy-plus#988]
**URL:** 
https://adhocracy.plus/werder-havel/projects/zukunftshaushalt/

**user:** 
admin

**expected behaviour:** 

- the numbers in the idea clusters should be readable and have a contrast ratio adhering to WCAG/ BITV recommendations
- The clusters should be easily visible on all colors of a map background

**behaviour:** 

- the numbers are hard to read even on a bright and high-contrast screen
- The clusters are hard to discern from the map background, especially on grey or green map areas

**important screensize:**
all sizes

**device & browser:** 
macOS Big Sur, Safari 14.0.1

**Comment/Question:** 

Screenshot?
<img width="853" alt="Bildschirmfoto 2020-12-01 um 10 50 31" src="https://user-images.githubusercontent.com/15341015/100724872-a3856600-33c3-11eb-948c-3ccb569dd34f.png">
